src/modules/10-common/RouteDefinitions.ts

```diff
@@ -861,7 +862,24 @@ const routes = {
       })
     }
   ),
-
+  toSecretDetailsRuntimeUsage: withAccountId(
+    ({
+      orgIdentifier,
+      projectIdentifier,
+      module,
+      secretId
+    }: Partial<ProjectPathProps & ModulePathParams & SecretsPathProps>) => {
+      const path = `resources/secrets/${secretId}/runtimeusage`
```

Comment:
Noticed we use hyphen-casing in URL paths.

```suggestion
      const path = `resources/secrets/${secretId}/runtime-usage`
```